src/sentry/static/sentry/app/components/stream/group.tsx

```diff
@@ -214,6 +225,42 @@ class StreamGroup extends React.Component<Props, State> {
     };
   }
 
+  renderReprocessingColumns() {
+    const {data} = this.state;
+    const {statusDetails, count} = data as GroupReprocessing;
+    const {info, pendingEvents} = statusDetails;
+    const {totalEvents, dateCreated} = info;
+
+    const remainingEventsToReprocess = totalEvents - pendingEvents;
+    const remainingEventsToReprocessPercent = percent(
+      remainingEventsToReprocess,
+      totalEvents
+    );
+
+    const value = remainingEventsToReprocessPercent || 100;
+
+    return (
+      <React.Fragment>
+        <StartedColumn>
+          <TimeSince date={dateCreated} />
+        </StartedColumn>
+        <EventsReprocessedColumn>
+          {!defined(count) ? (
+            <Placeholder height="17px" />
+          ) : (
+            <React.Fragment>
+              {`${totalEvents}/`}
```

Comment:
You could use the `<Count>` component, or `toLocaleString()` to format this better.